lib/racecar/consumer_set.rb

```diff
@@ -0,0 +1,90 @@
+module Racecar
+  class ConsumerSet
+    def initialize(config, logger)
+      @config, @logger = config, logger
+      @consumers = []
+      @consumer_iterator = [].cycle
+    end
+
+    def subscribe
+      raise ArgumentError, "Subscriptions must not be empty when subscribing" if @config.subscriptions.empty?
+      @consumers = @config.subscriptions.map do |subscription|
+        consumer = Rdkafka::Config.new(rdkafka_config(subscription)).consumer
+        consumer.subscribe subscription.topic
+        consumer
```

Comment:
Does an Rdkafka consumer not support multiple subscriptions?

Reply:
Ah, is it because rdkafka-ruby doesn't support per-subscription config? That's supported by the protocol though – does librdkafka not support that?

Reply:
It does, but not with different configuration options (like `auto.offset.reset`, which is quite handy to define in the handler for different topics).

Reply:
:+1:

Reply:
We'll see if we run into performance issues with many subscriptions then.

Reply:
That's correct, there can be only one config per client. This is a limitation imposed by the C lib.